src/styles/themes.js

```diff
@@ -108,8 +108,6 @@ export const HathorTheme = {
 const alertBaseColor = new HslColor('hsl(47, 100%, 62%)');
 export const AlertUI = {
   baseHslColor: alertBaseColor,
-  primaryColor: alertBaseColor.toString(),
-  light20Color: alertBaseColor.addLightness(20).toString(),
-  dark20Color: alertBaseColor.addLightness(-20).toString(),
-  dark40Color: alertBaseColor.addLightness(-40).toString(),
+  lightColor: alertBaseColor.addLightness(20).toString(),
+  darkColor: alertBaseColor.addLightness(-70).toString(),
```

Comment:
With this, the lightness will be 0, right? So it will be black, is that correct?

Reply:
We can use black to simplify, but this color is not black, it has 12 points of light. It keeps the yellow but with good contrast.

Reply:
It is correct now, sorry: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/371/commits/716ae6b0dc7aafb4bfe1748394b23ac7a6b38505